reposilite-plugins/webhook-plugin/src/main/kotlin/com/reposilite/plugin/webhook/listener/WebhookEvent.kt

```diff
@@ -0,0 +1,25 @@
+package com.reposilite.plugin.webhook.listener
+
+import com.reposilite.maven.Repository
+import com.reposilite.plugin.webhook.api.sendDiscordEmbed
+import com.reposilite.plugin.webhook.api.sendDiscordMessage
+import com.reposilite.storage.api.Location
+
+
+fun webhookEvent(by: String, gav: Location, repo: Repository) {
```

Comment:
It's not a listener, I think it should be moved to sth like `WebhookFacade.triggerWebhooks(params)`

Reply:
I don’t really understand how the Facade works

Reply:
Facade is just a name for class that exposes public functions to other domains (API). The difference between `Facade` and `Service` is only the visibility - `Service` is meant to use and handle internal implementation. You can find more about used architecture here:

* https://github.com/dzikoysk/reposilite/blob/main/.github/CONTRIBUTING.md